app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -13,8 +13,11 @@ AccountService = ($state, $auth, $modal, $http, $translate, ShortFormApplication
   Service.myApplications = []
   Service.createdAccount = {}
   Service.rememberedShortFormState = null
-  Service.accountError = {message: null}
-  Service.accountSuccess = {message: null}
+  Service.accountError =
+    message: null
```

Comment:
yeah! didn't totally refactor but this is definitely a start. 

and I agree, I figured "baby steps" and not reworking all the account messages at once, and let it use `message` for the previous features (sign in, create account, forgot password) without breaking anything. However yes that would probably be even better to consolidate all of that.